TTS/tts/models/base_tts.py

```diff
@@ -250,6 +251,14 @@ def get_sampler(self, config: Coqpit, dataset: TTSDataset, num_gpus=1):
             else:
                 weights = get_speaker_balancer_weights(data_items) * alpha
 
+        if getattr(config, "use_length_weighted_sampler", False):
+            alpha = getattr(config, "length_weighted_sampler_alpha", 1.0)
+            print(" > Using Length weighted sampler with alpha:", alpha)
```

Comment:
How does the sampler behaves if we change the order of dataset samples somewhere in the code? 

I am not sure if we keep the data order untouched in the TTSDataset

Reply:
The order of the data doesn't matter as the get_sampler function is called every time get_data_loader is called. Therefore, if we change the order of the data in the dataset, the weights will be recalculated.